build-system/tasks/runtime-test/index.js

```diff
@@ -647,7 +647,44 @@ async function runTests() {
   }
 }
 
+//TODO(estherkim): delete this file by the end of the week
```

Comment:
Come to think of it - not everyone who contributes to AMP does so on such a regular basis... I think we should leave this lying around a little longer than a week. Maybe 3 months?

Reply:
Sg. And the file goes, "Not today!"